README.md

```diff
@@ -31,104 +31,20 @@
 
 ##
 
-This crate is a Rust implementation of the primitives for creating and manipulating IPLD graphs that encode WNFS.
+This is a Rust implementation of [the WebNative FileSystem (WNFS) specification](https://github.com/wnfs-wg/spec). WNFS is a versioned content-addressable distributed filesystem with private and public sub systems. The private filesystem is encrypted so that only users with the right keys can access its contents. It is designed to prevent inferring metadata like the structure of the file tree. The other part of the WNFS filesystem is a simpler public filesystem that is not encrypted and can be accessed by anyone with the right address.
```

Comment:
minor: readme wise, should we move to have links at the bottom, e.g. makes the markdown a bit easier to read https://gist.githubusercontent.com/ashmoran/1181085/raw/ba256f98b5c14cd3feb0fffab7b2638a3ca7f377/README.markdown.

Reply:
this for all readmes (since we're chatting cleanup)